### ISSUE TITLE
Mixing settings

### ISSUE THREAD

**Issue by TalAmuyal** [extr0py/oni#225]
I enabled the default configuration of Oni in addition to my own nvim conf.
As far as I understand, Oni's configuration take precedence.
I came to this conclusion, as only some of my configurations are enabled, while others aren't.
Am I right? and if so, how can I add a Vim configuration that take precedence over both?

Ideally, I would like to use Oni's default configuration as a starting point, and add my own / override specific behavior.

For example, currently I can't use Oni as it forces space indentation while the project I work on has tab indentation.

**Comment by bert88sta** [comment]
In your config file for oni, have you set 

oni.loadInitVim: true

If you haven't, your init.vim is completely skipped over in terms of loading, and there might just be some happy overlap between yours and the default. If you're feeling brave, you can set both that setting and the default setting to true. I'll have to see the order the files are loaded in, as that would set the precedence for neovim.

**Comment by TalAmuyal** [comment]
Yes, my `~/.oni/config.json`:

```
{
    "oni.useDefaultConfig": true,
    "oni.loadInitVim": true,
    "editor.completions.enabled": true,
    "editor.fullScreenOnStart": true
}
```

If `oni.useDefaultConfig` is set to `false`, the BG becomes black and some other coloring degrade, but Oni behaives as I expect nvim to.
If `oni.useDefaultConfig` is set to `true`, Oni behaves as Oni usually does (like the white-space approach described in the previous post), but some of my configurations are still in place (like YCM).

From looking a bit around the code I found that in `browser/src/NeovimInstance.ts` line 367 there is a function called `startNeovim` that starts Neovim and either lets it load files as usual or denies the loading of the configuration using the `-u` flag.

So I assume that the configurations that are Oni's default are loaded afterward (using my common sense only, so I might be wrong) and thus take precedence by overriding previous settings.

Is there a current method of cooperating with it?

**Comment by extr0py** [comment]
Just added a long comment in #227 :) Thanks for all the details, it's very helpful!

Here's the TL;DR from the notes I put in the PR:
Does oni.useDefaultConfig set to false + setting the colorscheme in your init.vim give you the desired result? If so, I would lean towards that. And if that works, maybe we could update the documentation to be more clearer about the intended purpose / use of this setting?

**Comment by extr0py** [comment]
Looks like this was addressed by #256, so closing this one. Feel free to open a new issue if there are still any concerns. Thanks!
